web/src/components/people/ExternalInvite.tsx

```diff
@@ -0,0 +1,111 @@
+import Button from "@/components/ui/Button";
+import { useEffect, useState } from "react";
+import { Input, Alert, AlertMessage, Label } from "../shared/exports";
+import styles from "./ExternalInvite.module.css";
+import { postUsersInvite } from "@/openapi";
+import Loading from "@/components/ui/Loading";
+import Dialog from "../ui/Dialog";
+
+export default function ExternalInvite() {
+  const [buttonDisabled, setButtonDisabled] = useState(false);
+  const [isDialogVisible, setDialogVisible] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [email, setEmail] = useState("");
+  const [showSuccessMessage, setShowSuccessMessage] = useState(false);
+  const [showErrorMessage, setShowErrorMessage] = useState("");
+
```

Comment:
The current naming of `showErrorMessage` indicates it's a boolean (i.e. it is either shown or not shown). Calling it something like `errorMessage` is a bit closer to what the variable actually represents.